rootfs/lua/avaya_validate_path.lua

```diff
@@ -0,0 +1,43 @@
+
+local _M = {}
+
+-- alternatively: local lrucache = require "resty.lrucache.pureffi"
+local lrucache = require "resty.lrucache"
+local cache_ttl_seconds = 300 -- five minutes
```

Comment:
how often are these accounts updated? We could go a lot longer than 5 minutes and lookups are decently expensive. How about at least an hour?

Reply:
Sure I can change it. My thought was that with longer timeouts it could take a while before the cache is updated with the most recent data for an account(even if the account doesn't change too often).